src/virtualship/cli/commands.py

```diff
@@ -43,7 +53,16 @@ def init(path):
         )
 
     config.write_text(utils.get_example_config())
-    schedule.write_text(utils.get_example_schedule())
+
+    if mfp_file:
+        # Generate schedule.yaml from the MPF file
+        click.echo(f"Generating schedule from {mfp_file}...")
+        mfp_to_yaml(
+            mfp_file, str(path)
+        )  # Pass the path to save in the correct directory
```

Comment:
If `mfp_to_yaml` returned a string representing the yaml instead of doing the file modifications, then we can use `schedule.write_text(...)` like in the else clause. So something like

```py
if mfp_file:
    # don't repeat the comment here :) the log message is comment enough
    click.echo(f"Generating schedule from {mfp_file}...")
    schedule_body = mfp_to_yaml(
        mfp_file,
    )
else:
    schedule_body = utils.get_example_schedule()
schedule.write_text(...)

```